drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -589,6 +585,9 @@ static inline void qspi_fill_init_struct(nrfx_qspi_config_t *initstruct)
 	initstruct->phy_if.sck_freq = (INST_0_SCK_FREQUENCY > NRF_QSPI_BASE_CLOCK_FREQ) ?
 					      NRF_QSPI_FREQ_DIV1 :
 					      (NRF_QSPI_BASE_CLOCK_FREQ / INST_0_SCK_FREQUENCY) - 1;
+	if (INST_0_SCK_FREQUENCY/MHZ(8) > 1) {
```

Comment:
Wouldn't `(INST_0_SCK_FREQUENCY >= MHZ(16))` be clearer?

Reply:
Yes, it was like this, I converted to division to avoid checkpatch warning about constant on left side.

Reply:
```
WARNING:CONSTANT_COMPARISON: Comparisons should place the constant on the right side of the test
#67: FILE: drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c:588:
+	if (INST_0_SCK_FREQUENCY >= MHZ(16)) {

- total: 0 errors, 1 warnings, 118 lines checked
```
This is the error.